# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/sys-fcntl.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Some crap, like gpm.h does use this header.

  - Rene Rebe <rene@exactcode.de>

--- /dev/null	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.28/include/sys/fcntl.h	2005-02-07 12:34:33.300839392 +0100
@@ -0,0 +1,4 @@
+
+#warning "This is a non-standard header - use <fcntl.h> instead!"
+#include <fcntl.h>
+
